### PR TITLE
ListView: created list view with sample data

### PR DESCRIPTION
**Pull request by carizzuti**
I created a list view component with default/sample data using names

image

### CODE REVIEW

File: src/components/listView/ListView.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ListViewInfo } from './ListView.view';
+
+export const ListView = () => {
+    
+    const data = [{key: 'Charlie'}, {key: 'Michael'}, {key: 'Dulaya'},
+    {key: 'Leo'}, {key: 'Nicholas'}, {key: 'Miguel'}, {key: 'Rahmi'}];
+
```
Comment (raynesmike) [inline src/components/listView/ListView.js:8]: Looks great, im assuming we will be getting all this data from the backend

**Review by raynesmike — APPROVED** [review]
looks great with all the additions,

File: src/screens/dashboard/DashboardScreen.js [hunk 2]
```diff
@@ -2,15 +2,22 @@ import React from 'react';
 import {Icon} from '../../components/icon/Icon'
```
Comment (lasharela) [inline src/screens/dashboard/DashboardScreen.js:2]: Import from index

File: src/screens/dashboard/DashboardScreen.js [hunk 3]
```diff
@@ -2,15 +2,22 @@ import React from 'react';
...
 import {Text, View} from 'react-native';
 import {Button} from '../../components';
+import { ListView } from '../../components/listView/ListView';
```
Comment (lasharela) [inline src/screens/dashboard/DashboardScreen.js:5]: From index

**Review by lasharela — CHANGES_REQUESTED** [review]

**Review by mshago — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]
Looks okay. We'll put it together.

**Pull request merged**